pose_to_segments/bin.py

```diff
@@ -62,26 +63,30 @@ def get_args():
     parser.add_argument('-i', required=True, type=str, help='path to input pose file')
     parser.add_argument('-o', required=True, type=str, help='path to output elan file')
     parser.add_argument('--video', default=None, required=False, type=str, help='path to video file')
+    parser.add_argument('--subtitle', default=None, required=False, type=str, help='path to subtitle file')
```

Comment:
`subtitle` should be `subtitles` (same below)
(a subtitle refers to a single cue)